VideoCallKitQuickStart/ViewController+CallKit.swift

```diff
@@ -16,7 +16,9 @@ extension ViewController : CXProviderDelegate {
     func providerDidReset(_ provider: CXProvider) {
         logMessage(messageText: "providerDidReset:")
 
-        TVIAudioController.shared().stopAudio()
+        // AudioDevice is enabled by default
+        self.audioDevice.isEnabled = true
```

Comment:
Don't we want it disabled by default? We should encourage users to be really sure that no audio can possibly play before CallKit has done its magic.

Reply:
We can. I was thinking we will change it back to the default value after callkit use is done.

Reply:
I guess I don't feel that strongly about it. Lets keep what you have, its consistent with the voice QuickStarts.